cluster/examples/kubernetes/ceph/csi/example/rbd/configure-storageclass.bash

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -eux
+
+######################################
+# Example helper script to automate:
+# 1. Creating a CephBlockPool
+# 2. Verify existence of a kuberentes login on the Ceph cluster
+# 3. Fetches base64 encoded keys for admin and kubernetes logins on cluster and creates a Secret object with them
+# 4. Parses out the Cluster IPs for the Ceph mons  and creates a csi-rbd storageclass
+######################################
+
+<< EOF | kubectl create -f -
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: rbd
+  namespace: rook-ceph
+spec:
+  failureDomain: host
+  replicated:
+    size: 3
+EOF
+
+# Ensure that we have a kubernetes login configured on the ceph cluster
+(pod=$(kubectl get pod -n rook-ceph -l app=rook-ceph-operator  -o jsonpath="{.items[0].metadata.name}"); kubectl exec -ti -n rook-ceph ${pod} -- bash -c "ceph -c /var/lib/rook/rook-ceph/rook-ceph.config auth get-or-create-key client.kubernetes mon \"allow profile rbd\" osd \"profile rbd pool=rbd\"")
+
+# Retrieve the base64 encoded passwords for the admin and kubernetes users on the cluster and store them in env variables to use for storageclass config
+admin_key=$(pod=$(kubectl get pod -n rook-ceph -l app=rook-ceph-operator  -o jsonpath="{.items[0].metadata.name}"); kubectl exec -ti -n rook-ceph ${pod} -- bash -c "ceph auth get-key client.admin -c /var/lib/rook/rook-ceph/rook-ceph.config | base64")
+kubernetes_key=$(pod=$(kubectl get pod -n rook-ceph -l app=rook-ceph-operator  -o jsonpath="{.items[0].metadata.name}"); kubectl exec -ti -n rook-ceph ${pod} -- bash -c "ceph auth get-key client.kubernetes -c /var/lib/rook/rook-ceph/rook-ceph.config | base64")
+
+# Use the keys from above to create a secrets object on our Kubernetes cluster
+<< EOF | kubectl create -f -
```

Comment:
cat <<EOF | kubectl create -f -

Reply:
Same as above, so I guess folks need to decide what they want on this one.  Personally I always use the "cat<<EOF" format myself.

Reply:
Fails for me
<< EOF | kubectl create -f -
> apiVersion: ceph.rook.io/v1
> kind: CephBlockPool
> metadata:
>   name: rbd
>   namespace: rook-ceph
> spec:
>   failureDomain: host
>   replicated:
>     size: 3
> EOF
error: no objects passed to create

Reply:
Oh that's interesting!  I really wish this would've come up before squashing.  Ok, I'm going to change it back.  I would be curious to document what os and version you're on just in case the question comes up again. Thanks!